console/console-init/ui/src/Routes.tsx

```diff
@@ -39,5 +41,10 @@ export const AppRoutes = () => (
       getComponent={getConnectionDetail}
       exact={true}
     />
+    <LazyRoute
+      path="/iot-projects/:namespace/:projectname/:deviceid/:subList"
```

Comment:
please change the path to `/iot-projects/:namespace/:projectname/devices/:deviceid/:subList`